packages/SwingSet/src/vats/comms/parseRemoteSlot.js

```diff
@@ -7,7 +7,7 @@ import { assert, details } from '@agoric/assert';
 // message, and "-" when allocated by the sender of the message.
 
 export function parseRemoteSlot(s) {
-  assert(s, `${s}`, details`${s} is not a string`);
+  assert.equal(s, `${s}`, details`${s} is not a string`);
```

Comment:
This may now become `assert.typeof`... of course :)

Reply:
Already done ;)